src/DotLiquid.Tests/FilterTests.cs

```diff
@@ -261,5 +261,21 @@ public void TestContextFilter()
             _context.AddFilters(typeof(ContextFilters));
             Assert.AreEqual(" King Kong has 1000$ ", new Variable("var | bank_statement").Render(_context));
         }
+
+
+        [Test]
+        public void Truncate()
+        {
+            _context["var"] = "this is a longish string";

```

Comment:
Ideally cover the sample from the specification at https://shopify.github.io/liquid/filters/truncate/

{{ "Ground control to Major Tom." | truncate: 20 }}
{{ "Ground control to Major Tom." | truncate: 25, ", and so on" }}
{{ "Ground control to Major Tom." | truncate: 20, "" }}

Reply:
OK - will do